bin/yaml/libraries.yaml

```diff
@@ -36,7 +36,7 @@ libraries:
         - 4.0.0
         - 4.1.0
         - 5.0.0
-        - 5.1.0 
```

Comment:
My editor automatically remove this trailing whitespace. I can revert the change to this line if you want me to.

Reply:
its fine, dont worry about it